### DISCUSSION TITLE
GPUI Window Transparency

### DISCUSSION THREAD

**Discussion by jafioti** [zed-industries/zed#6734] [Feature Requests]
Is there a way to have semi-transparent / blurred windows in GPUI? I haven't seen any option for it in the WindowOptions. I know transparency can be a bit tricky so I understand if it isn't available yet, but it would be awesome to match the theme of many other macos apps that use blurred transparency.

**Comment by seradotflac** [comment]
NSVisualEffectView could be implemented into GPUI as a .blur/.vibrancy option on a div() and with it eventually added on Zed. Would be very fancy should it get implemented.

**Comment by tpisto** [comment]
This would be great indeed. There is currently an issue about that:
https://github.com/zed-industries/zed/issues/5040
